packages/frontend/src/components/Charts/ShortFundingChart.tsx

```diff
@@ -0,0 +1,168 @@
+import { memo } from 'react'
+import { Box, CircularProgress, createStyles, makeStyles, Tooltip } from '@material-ui/core'
+import InfoIcon from '@material-ui/icons/InfoOutlined'
+import dynamic from 'next/dynamic'
+import React, { useState } from 'react'
+import CustomSwitch, { SwitchItem } from '@components/CustomSwitch'
+import { useNormHistory } from '@hooks/useNormHistory'
+import { Tooltips } from '@constants/enums'
+import { graphOptions } from '../../constants/diagram'
+
+const Chart = dynamic(() => import('kaktana-react-lightweight-charts'), { ssr: false })
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    switchWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+      marginBottom: '16px',
+    },
+    legendBox: {
+      display: 'flex',
+      gap: '16px',
+      marginTop: '24px',
+      justifyContent: 'center',
+    },
+    legendContainer: {
+      display: 'flex',
+      gap: '5px',
+    },
+    iconWrapper: {
+      display: 'flex',
+      marginLeft: 6,
+    },
+  }),
+)
+
+const FundingChart = () => {
```

Comment:
Notice that the title says short funding chart, but we should show it both long and short when we add back the tabs, to be consistent with data on the current site

Reply:
not sure where the line color is, but we should change it to #70E3F6 for consistency with the new blue in the new designs